packages/devtools_app/lib/src/shared/side_panel.dart

```diff
@@ -272,7 +303,7 @@ class ReleaseNotesController {
   }
 
   void toggleReleaseNotesVisible(bool visible) {
-    _releaseNotesVisible.value = visible;
+    _isVisible.value = visible;
   }
```

Comment:
delete this method all together and have uses of it call toggleVisibility from the super class instead